infrastructures/kubernetes/src/main/java/org/eclipse/che/workspace/infrastructure/kubernetes/K8sInfraNamespaceWsAttributeValidator.java

```diff
@@ -76,7 +77,14 @@ public void validate(Map<String, String> attributes) throws ValidationException
                 + "')");
       }
 
-      namespaceFactoryProvider.get().checkIfNamespaceIsAllowed(namespace);
```

Comment:
Might be wrong, but according to the comment below it looks like idling is broken in the upstream also, no ?

Reply:
if this is the case it is just not clear for me why this error was not caught by the pre-release tests..

Reply:
@dmytro-ndp @rhopp could you please take a look? would be great to understand if the pre-release testing actually run the idling tests or this use-case is not tested atm?

Reply:
We talked about this and as far as I understood, this bug doesn't exhibit itself because in the pre-release tests, we use a namespace default that doesn't have `<username>` or `<userid>` placeholder.

Reply:
@metlos could you point me to the code which checks the namespace part and placeholders <username> / <userid> ? For now I do not fully understand how idling works and it looks like it works differently depending on the namespace strategies

Reply:
This is the method used during workspace update to check that the namespace contained in the attributes is valid: https://github.com/eclipse/che/blob/b2b6f5f488ea523bac0a9313f0f65a23ac98a16a/infrastructures/kubernetes/src/main/java/org/eclipse/che/workspace/infrastructure/kubernetes/namespace/KubernetesNamespaceFactory.java#L139

 Note that the current version of code in this branch calls this during the update only during a situation that I am not sure can happen even.